src/BulletListNodeSpec.js

```diff
@@ -34,15 +36,27 @@ const BulletListNodeSpec: NodeSpec = {
       },
     },
   ],
+
   toDOM(node: Node) {
     const {indent, listStyleType} = node.attrs;
     const attrs = {};
+
     if (indent) {
       attrs[ATTRIBUTE_INDENT] = indent;
     }
+
     if (listStyleType) {
       attrs[ATTRIBUTE_LIST_STYLE_TYPE] = listStyleType;
     }
+
+    let htmlListStyleType = listStyleType;
+
+    if (!htmlListStyleType || htmlListStyleType === 'disc') {
+      htmlListStyleType =
+        AUTO_LIST_STYLE_TYPES[indent % AUTO_LIST_STYLE_TYPES.length];
+    }
+
+    attrs.type = htmlListStyleType;
```

Comment:
bump

Reply:
Nope.

`type` is just for the presentation (the view), and spec only defines the data that reflects internal state of the document.

- `ATTRIBUTE_LIST_STYLE_TYPE`: the source of the truth of the list style defined at the spec.
- `type` the list type rendered. 

The difference matters because a list with list type of `decimal` or `null` may render diffrent list type (depends on the indentation level).

when user changes the indentation of a list, it should not mutate the internal state of the list type.